sdk/core/core-amqp/README.md

```diff
@@ -243,10 +94,6 @@ export DEBUG=azure:core-amqp:(error|warning),rhea-promise:error,rhea:events,rhea
     node your-test-script.js &> out.log
     ```
 
-# Next steps
```

Comment:
CI might complain of the missing section?

Reply:
Shoot, thought we already had an exception, added it now!